cron/worker/main.go

```diff
@@ -71,6 +72,16 @@ func processRequest(ctx context.Context,
 		return nil
 	}
 
+	exists, err = data.BlobExists(ctx, bucketURL2, filename)
+	if err != nil {
+		return fmt.Errorf("error during BlobExists: %w", err)
+	}
+	if exists {
```

Comment:
If this `exists` is true, that means the data-v1 was not written but data-v2 was. In that case, we shouldn't early return. Instead this should be:

```
if existsV1 && existsV2 {
 return ...
}
```